cmd/juju/crossmodel/find.go

```diff
@@ -18,18 +18,13 @@ Find which offered application endpoints are available to the current user.
 
 This command is aimed for a user who wants to discover what endpoints are available to them.
 
-options:
--o, --output (= "")
-   specify an output file
---format (= tabular)
-   specify output format (tabular|json|yaml)
-
 Examples:
    $ juju find-endpoints
    $ juju find-endpoints mycontroller:
    $ juju find-endpoints fred/prod
    $ juju find-endpoints --interface mysql --url fred/prod
    $ juju find-endpoints --url fred/prod.db2
```

Comment:
can we just have --offer, rather than both --offer and --url, and allow either type of value? i.e. --offer=db2 or --offers=fred/prod.db2

Reply:
The issue is that "db2" above may be parsed as a partial url and result in a model name. So there's no way to distinguish between whether the user wants to search for offers in a specified model vs specified offer names.